tests/components/homeassistant/test_scene.py

```diff
@@ -320,3 +320,12 @@ async def test_config(hass):
     no_icon = hass.states.get("scene.scene_no_icon")
     assert no_icon is not None
     assert "icon" not in no_icon.attributes
+
+
+def test_validator():
+    """Test validators."""
+    parsed = ha_scene.STATES_SCHEMA({"light.Test": {"state": "on"}})
+    assert len(parsed) == 1
+    assert "light.test" in parsed
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6184085)